stacks/spica/src/bucket/components/cel-editor/cel.syntax.ts

```diff
@@ -0,0 +1,139 @@
+export const configuration: import("monaco-editor-core").languages.LanguageConfiguration = {
+  // symbols used as brackets
+  brackets: [["{", "}"], ["[", "]"], ["(", ")"]],
+  // symbols that are auto closed when typing
+  autoClosingPairs: [
+    {open: "{", close: "}"},
+    {open: "[", close: "]"},
+    {open: "(", close: ")"},
+    {open: '"', close: '"'},
+    {open: "'", close: "'"},
+    {open: "(*", close: "*)"}
+  ],
+  // symbols that that can be used to surround a selection
+  surroundingPairs: [
+    {open: "{", close: "}"},
+    {open: "[", close: "]"},
+    {open: "(", close: ")"},
+    {open: '"', close: '"'},
+    {open: "'", close: "'"}
+  ]
+};
+
+export const language: any = {
+  defaultToken: "invalid",
+  tokenizer: {
+    root: [[/[{}]/, "delimiter.bracket"], {include: "common"}],
+
+    common: [
+      // identifiers and keywords
+      [
+        /[a-z_$][\w$]*/,
+        {
+          cases: {
+            "@keywords": "keyword",
+            "@default": "identifier"
+          }
+        }
+      ],
+      [/[A-Z][\w\$]*/, "type.identifier"], // to show class names nicely
+      // [/[A-Z][\w\$]*/, 'identifier'],
```

Comment:
We should remove or uncomment this line.

Reply:
i wanna keep this in case i need to reenable it for some reason